sample-app/screens/Home/HomeData.tsx

```diff
@@ -102,59 +102,73 @@ export const getStoredData = async (state: any): Promise<object> => {
     const value = await AsyncStorage.getItem(storageKey)
     if (value !== null) {
       const settings = JSON.parse(value)
-      var token = state.token
-      var secret = state.secret
+
       var configuration = state.configuration
-      let tokenValue = settings.list[0].data[3].value as string
-      if (tokenValue) {
-        token = tokenValue
-      }
-      let secretValue = settings.list[0].data[4].value as string
-      if (secretValue) {
-        secret = secretValue
-      }
 
       return {
+
+        isSandboxed: settings.list[0].data[0].value as boolean,
```

Comment:
Would be good in the future to define constants instead of guessing the position in the list